packages/app/src/components/modals/BaseModal.tsx

```diff
@@ -72,17 +67,16 @@ const BaseModal: FC<Props> = ({ children, ok, cancel, okEnabled = true }) => {
                     onClick={clickOk}
                   >
                     <div className="font-bold">Continue</div>{" "}
-                    <BiRightArrowAlt className="ml-2" size={24} />
-                  </button>
-                  <button
-                    type="button"
-                    className="flex items-center mt-3 w-full justify-center rounded-md  bg-danger px-4 py-2 text-base font-medium shadow-sm text-white hover:opacity-70 focus:outline-none focus:ring-2 focus:ring-offset-2 sm:col-start-1 sm:mt-0 sm:text-sm"
+                    <FiArrowRight className="ml-2 scale-150" />
+                  </Button>
+                  <Button
+                    variant="danger"
+                    className="flex items-center mt-3 w-full justify-center rounded-md  px-4 py-2 text-base font-medium shadow-sm text-white hover:opacity-70 sm:col-start-1 sm:mt-0 sm:text-sm"
```

Comment:
As discussed:
- Some classes are already in <Button>
- Some classes might should be added to <Button>, like shadow and focus:outline?
- Add a prop "size" to <Button> with possible values "md" and "sm", where "sm" has the padding and font you need here, md should be default